Source/Scene/processPbrMaterials.js

```diff
@@ -103,8 +124,20 @@ define([
         if (defined(pbrMetallicRoughness) && !useSpecGloss) {
             for (parameterName in pbrMetallicRoughness) {
                 if (pbrMetallicRoughness.hasOwnProperty(parameterName)) {
+                    var value = pbrMetallicRoughness[parameterName];
                     uniformName = 'u_' + parameterName;
-                    generatedMaterialValues[uniformName] = pbrMetallicRoughness[parameterName];
+                    generatedMaterialValues[uniformName] = value;
+
+                    if (parameterName.indexOf('Texture') >= 0 && defined(value.extensions) && defined(value.extensions.KHR_texture_transform)) {
```

Comment:
The extension should also be handled on spec gloss textures and the emission, occlusion, and normal textures.